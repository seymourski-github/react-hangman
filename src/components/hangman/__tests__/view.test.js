import React from 'react';
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import Hangman from '../view';

const props = {
  alphabet: ['a','b','c','d','e','f','g','h','i','j','k','l','m'],
  mystery: 'My secret word'
};

describe('<Hangman /> Component', function() {
  let component;
  let wrapper;

  beforeEach(() => {
    component = renderer.create(<Hangman {...props} />);
    wrapper = shallow(<Hangman {...props} />);
  });

  it("renders and matches our snapshot", () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
