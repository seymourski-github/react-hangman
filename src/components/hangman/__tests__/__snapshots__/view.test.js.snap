// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hangman /> Component renders and matches our snapshot 1`] = `
<div
  className="panel"
  id="hangman"
>
  <div
    className="panel-content"
  >
    <button
      className="button"
      disabled={false}
      name="My test word"
      onClick={[Function]}
      style={null}
      type="button"
    >
      My test word
    </button>
  </div>
</div>
`;
